.gitignore

```diff
@@ -9,7 +9,6 @@ _site
 docs/pfe.min*
 docs/assets/playgrounds/
 docs/assets/javascript/elements/*.js
-!docs/assets/javascript/elements/uxdot-example.js
```

Comment:
this is the build artifact, and should be ignored

Reply:
line 11 should be enough to ignore it.  the ! would tell it to not be ignored.